README.md

```diff
@@ -0,0 +1,53 @@
+## orbit-db-cache
+
+Isomorphic cache used by `orbit-db`, implemented using `level.js` (indexedDB) for the browser and `leveldown` (leveldb) for node.
+
+### Used in
+
+* [orbit-db](https://github/com/orbitdb/orbit-db)
+
+### API
+
+#### Public methods
+
+##### `open`
+
+Open cache store and make it available to be used (indexedDB or leveldb)
+
+##### `close`
+
+Close cache store and delete cached items
+
+##### `destroy`
+
+Completely remove an existing store
+
+##### `get`
+
+Get an item from the cache store
+
+##### `set`
+
+Set a value for a given key on the cache store
+
+##### `del`
+
+Remove a given key-value from the cache store
+
+### Properties
+
+#### `storage`
```

Comment:
I believe this is something of a left-over and a mistake on my part: `storage` doesn't necessarily be exposed through the public API (as far as I can tell) but should rather be a "private" member variable. I would leave this out (and perhaps add a note in the code that we should refactor it). What do you think?

Reply:
Sorry, `directory` and `storage` shouldn't be there, they're not public properties. I'll just remove them since `storage` is already private (underscore convention)